src/main/webapp/user-settings.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>User Settings</title>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="/css/main.css">
+	<script src="/js/navigation-loader.js"></script>
+	<script src="/js/user-settings.js"></script>
+</head>
+
+
+<body onload="addLoginOrLogoutLinkToNavigation(); addBasicLinkNavigation('stats.html', 'Stats Page'); addBasicLinkNavigation('feed.html', 'Public Feed');addBasicLinkNavigation('charts.html', 'Charts Page');addBasicLinkNavigation('map.html', 'Maps Page');addBasicLinkNavigation('user-map.html', 'User Map');
+  	addBasicLinkNavigation('user-settings.html', 'User Settings');">
+
+<nav>
+    <ul id="navigation">
+      <li><a href="/">Home</a></li>
+    </ul>
```

Comment:
add
<li class="dropdown">
        <a href="javascript:void(0)" class="dropbtn">More</a>
        <ul class="dropdown-content" id="dropdown-content"></ul>
      </li>
under the Home li element so the dropdown menu still appears